### PR TITLE
Add business logic to calculate + show RUNE price in header

### LINKED ISSUE #1328: ADD: RUNE Price
# Add RUNE Price on Header
Right next to the Da/Night Icon add `RUNE: $14.50` with grey pill shading. 

Get price from here: https://midgard.thorchain.info/v2/stats

image

image

### PR DESCRIPTION
**Pull request by veado**
Peek 2021-05-31 20-11

- [x] Calculate RUNE prices based on selected `PricePool`
- [x] Update i18n
- [x] Quick fix: Remove duplicated `getPoolPriceValue' function in `src/renderer/services/binance/utils.ts`

Close #1328

### CODE REVIEW

File: src/renderer/i18n/ru/common.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ const common: CommonMessages = {
   'common.pools': 'Пулы',
   'common.pool': 'Пул',
   'common.price': 'Цена',
+  'common.price.rune': 'RUNE price - RU',
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/i18n/ru/common.ts]: 'common.price.rune': 'Стоимость RUNE',

File: src/renderer/components/header/stats/HeaderStats.tsx [hunk 2]
```diff
@@ -1,30 +1,61 @@
...
       </Styled.Container>
-      <Styled.Container>
-        <Styled.SubLabel>Vol. (24hrs)</Styled.SubLabel>
+      {/* <Styled.Container>
+        <Styled.Title>{intl.formatMessage({ id: 'common.volume24' })}</Styled.Title>
         <Styled.Label>
           {formatAssetAmountCurrency({ amount: baseToAsset(volume24.amount), asset: volume24.asset, decimal: 0 })}
         </Styled.Label>
-      </Styled.Container>
+      </Styled.Container> */}
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/header/stats/HeaderStats.tsx:58]: have we decided not to show `volume24` value or this will be added with another PR ?

File: src/renderer/views/pools/Pools.types.ts [hunk 3]
```diff
@@ -17,11 +17,14 @@ export type PricePoolAssets = PricePoolAsset[]
 
 export type PricePoolCurrencyWeights = Record<string, number>
 
+// TODO (@asgdx-team) Move all PricePool* types into `src/renderer/services/midgard/types.ts`
 export type PricePool = {
   asset: PricePoolAsset
   poolData: PoolData
 }
 
+export type PricePool$ = Rx.Observable<PricePool>
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/views/pools/Pools.types.ts]: just can not see any mentions of this type. do we need this ?

**Review by thatStrangeGuyThorchain — DISMISSED** [review]
👌  just questions

File: src/renderer/components/header/stats/HeaderStats.tsx (hunk 2 above)
Comment (veado) [reply src/renderer/components/header/stats/HeaderStats.tsx:58]: Yep, it will be done by another PR.

File: src/renderer/views/pools/Pools.types.ts (hunk 3 above)
Comment (veado) [reply src/renderer/views/pools/Pools.types.ts]: Good find! _It's a stale type definition - I will remove it ..._
